cartographer_ros/cartographer_ros/assets_writer.h

```diff
@@ -58,6 +60,10 @@ class AssetsWriter {
       point_pipeline_builder_;
 };
 
+// Creates a FileWriterFactory which creates a FileWriter for storing assets.
+::cartographer::io::FileWriterFactory CreateFileWriterFactory(
```

Comment:
Why was that changed? Seemed perfectly fine to me.

Reply:
Okay, I thought it helps to communicate the independence from AssetsWriter class internals. I reverted back.